docs/_functions/global/DU.md

```diff
@@ -0,0 +1,31 @@
+---
+name: DU
+parameters:
+  - name
+  - modifiers...
+---
+
+`DU` allows updating existing domains added by `D`. It behaves the same way, however it is mandatory that the domain was already added. The first argument, the domain name,
```

Comment:
Suggested text:

`DU` adds records (and metadata) to a domain. The domain must have previously been defined by `D()`. `DU` behaves the same as `D()` in all other ways: The first argument is the domain name. See the documentation of `D` for further details.